server/src/concepts/user/repositories/commands.js

```diff
@@ -0,0 +1,63 @@
+import User from "../model/User";
+
+const createUser = (data) => {
+  const newUser = new User({
+    ...data,
+  });
+
+  try {
+    newUser.save();
+    return newUser;
+  } catch (err) {
+    console.log(err);
+    return "error";
+  }
+};
+
+const updateUser = () => {
+  User.updateOne({}, function (err, result) {
```

Comment:
https://mongoosejs.com/docs/api.html#model_Model.updateOne
I think updateOne should takes 3 parameters:
1. filter (object, key: value that you want to filter something, for example: name: "super_name")
2. data to set (object, key: value, values you want to set in database), I thinking what about '$set' beacuse I see examples where is using just plain object { fieldToUpdate: "new_value" } or with $set: { $set: { fieldToUpdate: "new_value" } }. In docs is without $set, so i would try with that.
3. Is callback function.
I think we can leave this callback syntax and do just `const response = await User.updateOne()` with these 2 parameteres without callback.